packages/client/src/forms/index.ts

```diff
@@ -441,6 +441,9 @@ export interface IFormFieldBase {
   tooltip?: MessageDescriptor
   validator: validators.Validation[]
   required?: boolean
+  // Whether or not to run validation functions on the field if it's empty
+  // Default false
+  validateEmpty?: boolean
```

Comment:
Thanks for the comment. Though the property was self-explanatory, it sure helps 👍🏼